src/main/java/io/stargate/sgv2/jsonapi/service/embedding/operation/JinaAIEmbeddingClient.java

```diff
@@ -83,7 +83,9 @@ public Uni<List<float[]>> vectorize(
                       && jae.getErrorCode() == ErrorCode.EMBEDDING_PROVIDER_TIMEOUT);
                 })
             .retry()
-            .withBackOff(Duration.ofMillis(requestProperties.retryDelayInMillis()))
+            // Jina has intermittent embedding failure, set a longer retry delay to mitigate the
+            // issue
+            .withBackOff(Duration.ofMillis(200), Duration.ofMillis(800))
```

Comment:
Could you use "retryDelayInMillis" instead of 200, and then 4x that instead of 800?